.jshintrc

```diff
@@ -13,5 +13,14 @@
   "quotmark": "single",
   "undef": true,
   "unused": true,
-  "strict": true
+  "strict": true,
+  "globals"   : {
+    /* MOCHA */
```

Comment:
I've tried it and it works. jshint allows comments See https://github.com/gruntjs/grunt-contrib-jshint/commit/95aee7ba76bfe8f04c2b23814d6c98e7e9d5b1b6